source/Depreciation.cpp

```diff
@@ -337,9 +339,8 @@ double Depreciation::Depreciate(int age) const
 	if(age >= MAX_AGE + GRACE_PERIOD)
 		return FULL_DEPRECIATION;
 	
-	double daily = pow(DAILY_DEPRECIATION, age - GRACE_PERIOD);
-	double linear = static_cast<double>(MAX_AGE - (age - GRACE_PERIOD)) / MAX_AGE;
-	return FULL_DEPRECIATION + (1. - FULL_DEPRECIATION) * daily * linear;
+	double exponent = exp((-STEEPNESS * ((age - GRACE_PERIOD) - SIGMOID_MIDPOINT))
+	return (CURVE_MAX / (1. + exponent)) + Y_INTERCEPT_CORRECTION
```

Comment:
Pretty sure that this equation doesn't match the one in the graph you showed.
```suggestion
	return -(CURVE_MAX / (1. + exponent)) + Y_INTERCEPT_CORRECTION
```